src/portal_visualization/builders/anndata_builders.py

```diff
@@ -71,6 +73,48 @@ def get_conf_cells(self, marker=None):
         cell_set_obs.append("leiden")
         cell_set_obs_names.append("Leiden")
         gene_alias = 'var/hugo_symbol' if 'var' in z and 'hugo_symbol' in z['var'] else None
+        if (gene_alias and marker is not None):
```

Comment:
`gene_alias` not a `bool` is probably want to check it explicitly against `None`.

Reply:
I have a bad habit of depending on truthy/falsy values owing to how much time I spend in JS land 😄 I'll make this condition more explicit:
```suggestion
        if (gene_alias is not None and marker is not None):
```